scripts/check_file_license_apache.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+FLAG=$1
+FILE=$2
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+TIMESCALE_LOCATION=`dirname $SCRIPTPATH`
+
+LICENSE_STRING=
+FIRST_COMMENT=
+
```

Comment:
I think you might be missing a type for scripts, where the comments are prefixed with `#`.

Reply:
The vast majority of things in scripts don't have a license header, so travis isn't checking it at the moment.